content/networking_sd/app_mesh/appmesh-implementation/mesh-frontend-app.md

```diff
@@ -38,6 +38,18 @@ cd ~/environment/ecsdemo-frontend/cdk
 cdk synth
 ```
 
+**Steps for those who encounter an error - Part 1
+If you see an error with the above command with a version mismatch first check your CDK CLI version. 
+```bash
+cdk --version
+```
+
+**Steps for those who encounter an error - Part 2
+Update the CDK CLI to the new version once you're satisfied that you're not introducing breaking changes to your other projects (outside of this workshop) before you upgrade the CDK CLI.
+```bash
+npm install -g aws-cdk@latest
```

Comment:
Can this be run as a pre requisite for the lab ? Instead of adding it here ? That way we will be proactive instead of reactive. 
Also, I tried adding @latest in a different lab and noticed that aws-cdk@2.25.0 was not compatible with a few modules. I had to fallback on aws-cdk@1.157.0 for that lab. 
Should we say aws-cdk@2.25.0  instead of aws-cdk@latest ? I am scared tomorrow, there might be a new version of cdk that might not be compatible with a different module and things would break.